at_secondary/at_persistence_secondary_server/test/hive_keystore_impl_test.dart

```diff
@@ -158,15 +159,55 @@ void main() async {
       var keyStore = keyStoreManager.getSecondaryKeyStore()!;
       var atData = AtData();
       atData.data = '123';
-      await keyStore.create('phone', atData,
+      await keyStore.create('phone.wavi@test_user_1', atData,
           sharedKeyEncrypted: 'abc', publicKeyChecksum: 'xyz');
-      var dataFromHive = await (keyStore.get('phone'));
+      var dataFromHive = await (keyStore.get('phone.wavi@test_user_1'));
       expect(dataFromHive?.data, '123');
       expect(dataFromHive?.metaData, isNotNull);
       expect(dataFromHive?.metaData!.sharedKeyEnc, 'abc');
       expect(dataFromHive?.metaData!.pubKeyCS, 'xyz');
     });
 
+    test('test create key without namespace - keystore put', () async {
```

Comment:
I have written a test in the past with all of the valid and invalid keys, I will point to you that file so that those keys can be reused.